var/spack/repos/builtin/packages/ncbi-rmblastn/package.py

```diff
@@ -26,11 +26,102 @@
 
 
 class NcbiRmblastn(AutotoolsPackage):
-    """RMBlast search engine for NCBI"""
+    """RMBlast search engine for NCBI."""
 
     homepage = "https://www.ncbi.nlm.nih.gov/"
     url      = "ftp://ftp.ncbi.nlm.nih.gov/blast/executables/rmblast/LATEST/ncbi-rmblastn-2.2.28-src.tar.gz"
 
     version('2.2.28', 'fb5f4e2e02ffcb1b17af2e9f206c5c22')
+    version('2.6.0', 'c8ce8055b10c4d774d995f88c7cc6225',
+            url='ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.6.0/ncbi-blast-2.6.0+-src.tar.gz')
+
+    patch('isb-2.6.0+-changes-vers2.patch', when='@2.6.0', level=1)
```

Comment:
can you add a comment what the patch is doing? And why it is applied unconditionally

Reply:
This is the patch that makes this rmblasn. The program is basically just a patch to the regular blastn to enable features that repeatmasker needs (which is just a handful of small behavior changes and more or less renaming the executable). http://www.repeatmasker.org/RMBlast.html